tests/performance/url_hits.xml

```diff
@@ -32,6 +31,12 @@
            </values>
        </substitution>
     </substitutions>
-
     <query>SELECT count() FROM hits_100m_single WHERE NOT ignore({func}(URL))</query>
+
+    <!-- firstSignificantSubdomain/firstSignificantSubdomainCustom -->
```

Comment:
(cutTo)firstSignificantSubdomain(URL) is already tested by the above query, so I think we can leave just the Custom versions here, and use the same hits_100m_single table, for easier comparison.

Reply:
I don't see any point in comparing those functions with multiple threads, hence they were duplicated with max_threads=1 and smaller hits table.

And I decided to leave original queries as-is for now to avoid changes in perf tests